### PR TITLE
setup homepage

### PR DESCRIPTION
**Pull request by Shailseen**
**Making components :**
1. Navbar
2.  Carousel
3.  Poster
4. Arrival
5. Footer
image
image

### CODE REVIEW

File: src/components/arrival/Arrival.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                <img class="img-responsive with-border" src={item.image}/>
+              </div>
+              <div class="card-description-container">
+              <div class="text-container text-grid">
+                <h3>{item.productName}</h3>
+                <p>{item.productDescription}</p>
+                <p class="price-card"><span class="original-price">{item.price}</span> <span class="price-discount">{item.discountedPrice}</span> <span class="price-percantage">{item.percantageDiscount}</span></p>
+              </div>
+              <div class="card-button-container button-grid">
+                <button class="button-style-none solid-button" onclick="location.href='/pages/homepage/product-page.html'">Explore</button>
```
Comment (ishitakeshawani) [inline src/components/arrival/Arrival.jsx]: You can use Link or useNavigate here.
for reference https://stackoverflow.com/questions/50644976/react-button-onclick-redirect-page

File: src/components/carousel/Carousel.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import axios from "axios";
+import "./carousel.css";
+const Carousel = () => {
+  const [categories, setCategories] = useState([]);
+
+  const fetchCategories = async () => {
+    try {
+      const response = await axios.get(`/api/categories`);
+      setCategories(response.data.categories);
+      console.log(response.data.categories);
```
Comment (ishitakeshawani) [inline src/components/carousel/Carousel.jsx]: Remove this console log.

File: src/components/navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import "./navbar.css";
+const Navbar = () => {
+    return(
+    <nav className="navbar-container box-shadow">
+    <div className="navbar">
+      <div className="brand-name">
+        <a className="text-style-none font-lg" href="/pages/homepage/homepage.html"
```
Comment (ishitakeshawani) [inline src/components/navbar/Navbar.jsx:7]: use link here

File: src/components/index.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import { Navbar } from "./navbar/Navbar";
+import { Carousel } from "./carousel/Carousel";
```
Comment (ishitakeshawani) [inline src/components/index.js]: You can just export and use it.
export { Navbar } from "./navbar/Navbar";

**Review by ishitakeshawani — COMMENTED** [review]
added some comments.
It will be easy if you can add a screenshot of it.

**Comment by Shailseen** [comment]
> added some comments. It will be easy if you can add a screenshot of it.

done

File: src/components/index.js (hunk 4 above)
Comment (Shailseen) [reply src/components/index.js]: done

File: src/components/carousel/Carousel.jsx (hunk 2 above)
Comment (Shailseen) [reply src/components/carousel/Carousel.jsx]: okay

File: src/backend/db/categories.js [hunk 5]
```diff
@@ -1,4 +1,11 @@
 import { v4 as uuid } from "uuid";
+// import {image18} from "../../assets/index";
```
Comment (Aditi0505) [inline src/backend/db/categories.js]: Please remove if not needed.

File: src/components/carousel/carousel.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
+.menu-container{
+    display: flex;
+    overflow: auto;
+    overflow-y: hidden;
+    /* flex-wrap: wrap; */
```
Comment (Aditi0505) [inline src/components/carousel/carousel.css]: Remove comment if not necessary

File: src/components/footer/Footer.jsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+      </div>
+      <div class="icon-container">
+        <a class="footer-icon" href="">
+          <i class="fab fa-github"></i>
+        </a>
+        <a class="footer-icon" href="">
+          <i class="fab fa-twitter"></i>
+        </a>
+        <a class="footer-icon" href="">
+          <i class="fab fa-linkedin-in"></i>
```
Comment (Aditi0505) [inline src/components/footer/Footer.jsx]: Provide links here in href and you can make a common component for this t avoid code repetition.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions for you to look at. Rest looks good to me.

File: src/components/footer/Footer.jsx (hunk 7 above)
Comment (Shailseen) [reply src/components/footer/Footer.jsx]: done

**Pull request merged**